charts/fe-nextjs/templates/proxy.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if .Values.staticProxy.enabled }}
+{{- $fullName := include "app.fullname" . -}}
+{{- $svcPort := .Values.staticProxy.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ $fullName }}-static-proxy
+  labels:
+    {{- include "app.proxyLabels" . | nindent 4 }}
+  {{- with .Values.staticProxy.service.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  type: ExternalName
+  externalName: {{ .Values.staticProxy.service.externalName | quote }}
+  {{- with .Values.staticProxy.service.clusterIP }}
+  clusterIP: {{ . }}
+  {{- end }}
+  ports:
+    - name: http
+      port: {{ $svcPort }}
+      {{- if and (or (eq .Values.staticProxy.service.type "NodePort") (eq .Values.staticProxy.service.type "LoadBalancer")) .Values.staticProxy.service.nodePort }}
+      nodePort: {{ .Values.staticProxy.service.nodePort }}
+      {{- end }}
+      targetPort: {{ .Values.staticProxy.service.targetPort }}
+      protocol: TCP
+      {{- if semverCompare ">=1.20-0" .Capabilities.KubeVersion.GitVersion }}
+      appProtocol: http
+      {{- end }}
+  selector:
+    {{- include "app.proxySelectorLabels" . | nindent 4 }}
+---
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}-static-proxy
```

Comment:
as it's the only ingress of the chart, why adding to the name "static-proxy"?

Reply:
it is not.. https://github.com/casavo/charts/pull/1/files#diff-d24f275219f1bc1a6bca2abe899c83748039de12d81893c2692889d0e638ce9a